devops/helm/crunchy-postgres/values.yaml

```diff
@@ -0,0 +1,101 @@
+fullnameOverride: simsdb
+appuser: app-database-user
```

Comment:
About having this user created, is it a final and required decision needed for the upcoming DB restore plan?
The secondary possible plan of having the user created manually in the new DB is no longer being considered?

Reply:
no this creates the user with required permissions and mimic the one we have in out existing database.